### ISSUE TITLE
Aseprite

### ISSUE THREAD

**Issue by Cequallium** [catppuccin/catppuccin#201]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Comment by DakshG07** [comment]
Would be cool :/

**Comment by skinatro** [comment]
are libresprite ones compatible?
i can try making something for libresprite ig

**Comment by Cequallium** [comment]
yes i believe so

**Comment by Cequallium** [comment]
i'm currently editing the using the dase theme sheet as a base with some modifications, let me know if everyone's okay with that theme.

**Comment by Vixeliz** [comment]
Are you still working on this? If so any repository to check out? I'm interested in helping out

**Comment by Cequallium** [comment]
Yes! I am, although its stagnated a bit recently.. I need help making a build script to generate all the themes from a base theme, The repo I'm working in is currently private since its nowhere near ready to publish, also sorry for the late response.

**Comment by Cequallium** [comment]
The mocha-mauve version!
![image](https://user-images.githubusercontent.com/85212821/189264798-7d905af1-0018-45a5-b822-b943b3845950.png)

**Comment by Vixeliz** [comment]
Awesome! Unfortunately some stuff has come out so I can't help anymore at the moment. But awesome work.

**Comment by skinatro** [comment]
hey you can try out using [my script](https://github.com/skinatro/theme-tool)
sorry i have been busy  with exams, so i havent added a proper readme  yet

**Comment by Cequallium** [comment]
omg thanks i've got 3 out of the 4 themes done after using your script, its really convenient.

**Comment by Vixeliz** [comment]
No rush but I now have some more time and was wondering if the repo is public yet?
